package.json

```diff
@@ -36,6 +36,7 @@
         "lru-cache": "7.10.1",
         "next": "12.1.0",
         "next-seo": "5.4.0",
+        "pino": "^8.0.0",
```

Comment:
We are trying to keep all dependencies pinned to a specific version, so if we could do that with pino as well that would be great.